src/fragments/lib/interactions/js/getting-started.mdx

```diff
@@ -4,7 +4,58 @@ AWS Amplify Interactions category enables AI-powered chatbots in your web or mob
 
 AWS Amplify implements [Amazon Lex](https://aws.amazon.com/lex) as the default chatbots service. Amazon Lex supports creating conversational bots with the same deep learning technologies that power Amazon Alexa.
 
-## Create new chatbot
+<Callout>
+  The recommended approach is to begin your new App integration with AWS LexV2, as the default module exports are associated with AWS LexV2 APIs. Instructions for AWS LexV1 setup can be found in the following section.
+</Callout>
+
+## Setup AWS LexV2 bot
+You can create Amazon Lex V2 chatbot in Amazon Lex console. To create your bot, follow the steps shown in [Amazon Lex V2 Developer Guide](https://docs.aws.amazon.com/lexv2/latest/dg/getting-started.html).
+
+![Interactions](/images/interactions_lex_v2_console_edit_bot.png)
+
+With manual setup, you also need to add AWS Lex V2 API permissions to IAM roles and bot details to configure your app.
+
+For adding IAM permissions, find you IAM role and attach the policy below (*remember to replace the template with real value*):
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "lex:RecognizeText",
+        "lex:RecognizeUtterance"
+      ],
+      "Resource": "arn:aws:lex:<Region>:<Account>:bot-alias/<BotId>/<BotAliasId>"
+    }
+  ]
+}
+```
+
+```javascript
+import { Amplify } from 'aws-amplify';
+import awsconfig from './aws-exports';
```

Comment:
I think we might be using
```
import amplifyconfig from './amplifyconfiguration.json';
```
in docs now

Reply:
oh, nice. we are align with other platforms. will update it.